components/script/dom/gpudevice.rs

```diff
@@ -86,6 +86,29 @@ pub struct GPUDevice {
     valid: Cell<bool>,
 }
 
+pub enum PipelineLayout {
+    Implicit((PipelineLayoutId, Vec<BindGroupLayoutId>)),
```

Comment:
```suggestion
    Implicit(PipelineLayoutId, Vec<BindGroupLayoutId>),
```

The `Implict` variant is already a tuple-like structure, so having a nested tuple is redundant.